packages/integrations/components/integrationsManager/index.js

```diff
@@ -2,62 +2,52 @@ import React from 'react';
 import PropTypes from 'prop-types';
 // User Config.
 import userConfig from '@irvingjs/irving.config';
-// Redux.
-import { connect } from 'react-redux';
-import { actionHydrateComponents } from '../../actions';
 // Component Map.
 import componentMap from './componentMap';
 
 /**
  * Integrations Manager.
  *
  * A component that consumes a `irving/integrations` configuration from the
- * REST API, Irving config, or persisted Redux state slice and renders the
- * corresponding integrations on the client.
+ * REST API and renders the corresponding integrations on the client.
  *
  * @param {Object}   props - All props.
  * @param {Object}   props.integrations - Config object for active integrations.
- * @param {boolean}  props.isHydrated - Has the component map already been hydrated?
- * @param {function} props.hydrateComponents - Redux action for updating the `componentMap` in the state tree.
- * @param {Object}   props.hydratedComponents - The existing component key/prop pairs.
  */
-const IntegrationsManager = ({
-  integrations,
-  isHydrated,
-  hydrateComponents,
-  hydratedComponents,
-}) => {
-  // This check is run on server-side renders to ensure that the component hydration
-  // is ran a single time. After hydration, `isHydrated` is set to `true` and will
-  // only be re-run if the component props change during the component's lifecycle.
-  if (false === isHydrated) {
-    // Retrieve the keys available for render in the component map.
-    const keyMap = Object.keys(integrations);
+const IntegrationsManager = ({ integrations }) => {
+  // Test for empty objects.
+  const isEmpty = (obj) => (
+    0 === Object.keys(obj).length && obj.constructor === Object
+  );
+
+  // Bail early if no integrations exist.
+  if (isEmpty(integrations)) {
+    return null;
+  }
 
-    // Create an array for hydrated component keys and props.
-    const components = [];
+  // Retrieve the keys available for render in the component map.
+  const keyMap = Object.keys(integrations);
 
-    componentMap.forEach((component) => {
-      // Get the component's key.
-      const { key } = component;
+  // Create an array for hydrated component keys and props.
+  const components = [];
 
-      if (keyMap.includes(key)) {
-        // Retrieve the key/value pairs set for the integration and convert them into props.
-        // If a user config override exists, use it instead.
-        let props;
-        if ('integrations' in userConfig && key in userConfig.integrations) {
-          props = userConfig.integrations[key];
-        } else {
-          props = integrations[key];
-        }
-        // Update the item at the current index with the transformed component.
-        components.push({ key, props });
-      }
-    });
+  componentMap.forEach((component) => {
```

Comment:
This can probably further be refactored to avoid the need for this mutated integrations array, but was trying to avoid too many logic changes so this doesn't introduce regressions.